config/coordinator.sample.json

```diff
@@ -14,55 +14,63 @@
     "Queries": [
       {
         "Description": "Percentage of HTTP 5xx server errors",
+        "Legend": "Percentage of requests responded from the server with a 5xx status code",
```

Comment:
The legends are very close to the description. In any case, they should be one or two words at max. I'll suggest values like "percent" or "duration".

Reply:
> The legends are very close to the description.

Yeah, the thing is that we're using `Description` like a title or name. I thought about changing Description to Title, thoughts?

> In any case, they should be one or two words at max. I'll suggest values like "percent" or "duration".

Do you feel strongly about them being so short? I think I like a very descriptive legend, but maybe that's just me.

Reply:
> Do you feel strongly about them being so short? 

I do, but there is a functional reason for it. Labels should be short because if there are tens of them, then others become hard to read. In this case, there's just one, but I'd like to maintain the same convention.

I will defer to you for a final decision :)

Reply:
Indeed, my reasoning here was that in this case there is only one, but I'll buy the consistency argument :)

Reply:
Done in https://github.com/mattermost/mattermost-load-test-ng/pull/694/commits/29dc25fe01884c77454003761beb828e3b2f9fec